discovery/seata-discovery-raft/src/main/java/io/seata/discovery/registry/raft/RaftRegistryServiceImpl.java

```diff
@@ -470,6 +460,11 @@ public List<InetSocketAddress> lookup(String key) throws Exception {
                 if (CollectionUtils.isEmpty(list)) {
                     return null;
                 }
+                try {
+                    refreshToken(queryHttpAddress(clusterName, key));
+                } catch (RetryableException e) {
```

Comment:
```suggestion
                } catch (Exception e) {
```
I think anytime an exception occurs, the app should just be prevented from continuing to launch